### ISSUE TITLE
Don't prompt to "Add Config" when deploying .NET Core project with Local Git

### ISSUE THREAD

**Issue by XofoSol** [microsoft/vscode-azureappservice#1185]
Repro steps:

1. Do a commit in your local git
2. Select Local git as your deployment source
3. Do the deploy to webapp.
4. VSCode tells you that you need a new configuration to do the deploy (it doesn't matter if you had already done another deploy before)
6. Pum.  Error.

Action: appService.Deploy
Error type: Error
Error Message: Cannot read property 'constructor' of undefined

Version: 0.16.0
OS: darwin
Product: Visual Studio Code
Product Version: 1.38.1
Language: es

<details>
<summary>Call Stack</summary>

```
Object.<anonymous> extension.bundle.js:323:357962
a extension.bundle.js:323:357598
process._tickCallback next_tick.js:68:7
```

</details>

**Comment by XofoSol** [comment]
Update:
VSCode is creating a settings.json file with this info:
{
"appService.defaultWebAppToDeploy": "/subscriptions/{{subscriptionId}}/resourceGroups/{{ResourceGroup}}/providers/Microsoft.Web/sites/{{WebAppName}}",
    "appService.deploySubpath": "{{ProjectName}}/bin/Release/netcoreapp2.2/publish",
    "appService.preDeployTask": "publish-release"
}

If I delete this data or replace it with an empty json (like {}) it works again.

**Comment by EricJizbaMSFT** [comment]
@XofoSol as a workaround, you can try disabling the warning by setting `appService.showPreDeployWarning` to `false`

**Comment by XofoSol** [comment]
Nop, it does not work. Because is not a warning, is an error.

**Comment by EricJizbaMSFT** [comment]
Sorry I wasn't clear - I'm talking about this warning:
![Screen Shot 2019-09-20 at 1 18 28 PM](https://user-images.githubusercontent.com/11282622/65356387-37955400-dba9-11e9-9eb8-5f76830f49ec.png)

If you click "Add Config" it adds the settings you mentioned ("appService.deploySubpath" and "appService.preDeployTask"). I believe those settings are causing your error, so you'll want to remove them and set `appService.showPreDeployWarning` to `false` to prevent adding that config again

**Comment by XofoSol** [comment]
Ok, Thank you very much.  
Apparently it works.  But, like you said, you have to add this configuration in the file only after the first fail.

**Comment by EricJizbaMSFT** [comment]
Yeah that's only meant as a temporary workaround to make sure you're unblocked.

This is definitely a bug that we'll look into fixing soon
